demo/src/test/java/org/junitpioneer/jupiter/ReportEntryExtensionTests.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016-2021 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestReporter;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class ReportEntryExtensionTests {
+
+	// tag::no_report_entry[]
```

Comment:
Are the `[]` needed? So ceremonial... (in a bad way)

Reply:
Unfortunately, yes. Non-avoidable. :(